glide.lock

```diff
@@ -202,7 +202,7 @@ imports:
 - name: github.com/NYTimes/gziphandler
   version: 56545f4a5d46df9a6648819d1664c3a03a13ffdb
 - name: github.com/openshift/api
-  version: fe0c0ad070ce15d30b0a6dd20adb0ef883efe11a
+  version: e8532c51d46fdc99060e3357da9282a3c588ee94
```

Comment:
sha is almost a month old - did you clear your glide cache?

Reply:
must not have, updating.